docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -139,13 +139,17 @@ Preparing For a Release
 
       git push --set-upstream origin $(git rev-parse --abbrev-ref HEAD)
 
-#. Ensure all the required tests pass on BuildBot:
+#. Ensure all the required tests pass on Jenkins:
 
-   Pushing the branch in the previous step should have started a build on BuildBot.
-   If not, you can force a build by logging in to BuildBot, entering the release branch name in to the box at the top right and clicking the ``Force`` button.
+   To run the tests on `Jenkins`_, first run `setup_ClusterHQ-flocker-release <http://ci-live.clusterhq.com:8080/job/setup_ClusterHQ-flocker-release/>`_ using the release branch as the parameter to the job.
```

Comment:
Use double-backquotes around `setup_ClusterHQ-flocker-release` for consistency with comments below.  This may affect the URL generation, but it is probably not needed since you have the link to Jenkins on the same line.